### ISSUE TITLE
10 years of dat

### ISSUE THREAD

**Issue by ninabreznik** [dat-ecosystem/organization#119]
`@TODO`

- [x] research core events - Alex
- [x] watch old interviews and talks - Alex
- [ ] write and design:
    - [ ] dat 10 year anniversary post

---

`## Info`

Budget: 
- $400 ??

**Comment by millette** [comment]
Adding a note for myself @millette so I can, depending on the timeframe, work on a simple (and visual) timeline to help get this done.

**Comment by ninabreznik** [comment]
Hi, @millette this sounds great. @serapath has gone through all the old posts and talks and has some ideas about the structure. So maybe he can share this with you and you could help writing it down in a nice and understandable format.

**Comment by serapath** [comment]
oh sweet.
hey @millette , nice to see you around again 😊

There is a new website in the making, as you might have seen in github issue activity.
It will include a searchable timeline.

This task here is meant as an epic blog post, including maybe short interviews with the humans of dat through the ages 😁

Anyway, referencing/linking lots of of important documents/videos/etc... and structuring it by year and episode of dat, including the version of the dat logo at that time.

the blog and all the wayback machine versions of all website and conference videos or talks can be put into a timeline.


I started this process already and have been through hours, probably days of video material.

The hypothesis is, that while you cant watch gras grow, going through the material, you can all of a sudden - in retrospect - see how the ideas around dat shaped.


Yet another form of manifestation is the `dat-ecosystem-archive` which now has most l, but not all of the historic activity and we plan to reach out to more folks to make it more complete.

Basically the reoccuring and changed IDEAS found in all those dat modules is another description of what dat was trying to do.

we hope the epic probably multi chapter blog post can tell the story to capture what dat was or wanted to become from multiple perspectives so it can guide dat in the future, to maybe focus again on building out that vision, but this time being conscious about it and goal focused to finally make it happen.

 Back then, ambitions where large and the problems seemed tiny. Turned out building solid stable p2p infrastructure is a gargantuan challenge, but 10 years later we are mostly there, so we can probably give EVERYTHING that has been tried another shot for this time to succeed 🙂

The 10 year aniversary post imho could be the most important piece of work of the entire moving forward grant to ignite the next phase of dat 🔥🔥🔥🌈🚀

...

maybe i'm already so deep in, but we could potentially structure and split the work.
What is your inspiratiin for how this could be done and shaped?

**Comment by serapath** [comment]
oh sweet.
hey @millette , nice to see you around again 😊

There is a new website in the making, as you might have seen in github issue activity.
It will include a searchable timeline.

This task here is meant as an epic blog post, including maybe short interviews with the humans of dat through the ages 😁

Anyway, referencing/linking lots of of important documents/videos/etc... and structuring it by year and episode of dat, including the version of the dat logo at that time.

the blog and all the wayback machine versions of all website and conference videos or talks can be put into a timeline.


I started this process already and have been through hours, probably days of video material.

The hypothesis is, that while you cant watch gras grow, going through the material, you can all of a sudden - in retrospect - see how the ideas around dat shaped.


Yet another form of manifestation is the `dat-ecosystem-archive` which now has most l, but not all of the historic activity and we plan to reach out to more folks to make it more complete.

Basically the reoccuring and changed IDEAS found in all those dat modules is another description of what dat was trying to do.

we hope the epic probably multi chapter blog post can tell the story to capture what dat was or wanted to become from multiple perspectives so it can guide dat in the future, to maybe focus again on building out that vision, but this time being conscious about it and goal focused to finally make it happen.

 Back then, ambitions where large and the problems seemed tiny. Turned out building solid stable p2p infrastructure is a gargantuan challenge, but 10 years later we are mostly there, so we can probably give EVERYTHING that has been tried another shot for this time to succeed 🙂

The 10 year aniversary post imho could be the most important piece of work of the entire moving forward grant to ignite the next phase of dat 🔥🔥🔥🌈🚀

...

maybe i'm already so deep in, but we could potentially structure and split the work.
What is your inspiratiin for how this could be done and shaped?

**Comment by serapath** [comment]
@millette 
It kinda makes me happy to see familiar folks still lurking around 😊

afaik you made 
dat shell, deploy dat, dat resolver
and you could transition them to the dat-ecosystem-archive.
you have a few contributiins to mafintosh, okdistribute, mauve, ... etc... which are also not yet in the dat-ecosystem archive and maybe getting long timers historic take on dat could also be an opportunity to reach out and bring those modules into the archive too?


https://github.com/millette/dat-shell
https://github.com/millette/dat-resolver
https://github.com/millette/dat-store
https://github.com/millette/dat-archive-web
https://github.com/millette/deploy-dat
https://github.com/millette/pauls-dat-api
https://github.com/millette/dat-node
https://github.com/millette/awesome-dat
https://github.com/millette/dat-link-resolve
https://github.com/millette/dat-encoding
https://github.com/millette/dat-test
https://github.com/millette/webdb
https://github.com/millette/dat-dns
https://github.com/millette/dat-doctor
https://github.com/millette/hyperhealth

**Comment by millette** [comment]
> maybe i'm already so deep in, but we could potentially structure and split the work.
> What is your inspiratiin for how this could be done and shaped?

 @serapath Call me flabbergasted by the scope you've described :-)

I was thinking mostly 10-20 items per year as an overview, with the bit of time I have available.

I contributed very little, all over the place ;-) I <del>will transfer</del> <ins>transferred</ins> dat-shell to the [dat-ecosystem-archive](https://github.com/dat-ecosystem-archive) <del>as soon as I figure out how :-)</del> <ins>via your account since I can't transfer directly to the org.</ins>

**Comment by serapath** [comment]
The website has a timeline section.
Previous websites on wayback machine had timelines too.
There is a document with some historic events written down.
We would like to capture things in as much detail as possible and make it filterable/searchable.

If you have any items you think are missing it would be great to add them.
Regardless of what, maybe recorded conference talks, maybe projects which launched, released, shut down and where using dat, maybe modules launched and archived. The dat-ecosystem archive  and all the archived modules of course also have initial commits and a date when activity stopped ...all these things could be timeline entries.

oh, here is the hackmd.
https://hackmd.io/cuolp56RRkmMNqr_6fIW1A

Basically, just randomly adding material to be included in the history would be great.
Ideally we end up at some point with a history that includes absolutely everything :-)
Or at least have all the relevant stuff covered.

**Comment by serapath** [comment]
@millette 
The more personal stories we gather in as much detail as possible, the better.

Would you mind sharing your personal story with dat and all the history and how you perceive(d) it over time?

**Comment by millette** [comment]
Sounds good, but it will have to wait 10 days or more, I'm going on a little vacation :-)
